app/src/main/java/com/team41/boromi/FragmentFactory.java

```diff
@@ -0,0 +1,20 @@
+package com.team41.boromi;
+
+import android.os.Bundle;
+import androidx.fragment.app.Fragment;
+import com.team41.boromi.book.GenericListFragment;
+
+public class FragmentFactory {
+
+  public static Fragment createFragment(Class<? extends Fragment> c, Bundle bundle)
```

Comment:
Unhandled exceptions, where are these going to be picked up?

Reply:
in PageAdapter